operator/roles/csiscaleoperator/templates/cluster-role-attacher.yaml.j2

```diff
@@ -52,3 +52,14 @@ rules:
   verbs:
   - patch
 
+{% if "apps.openshift.io" not in api_groups and csipspname is defined %}
```

Comment:
I think csipspname is defined will always return true because of default.

Reply:
resolved in new change